Travel Scheduler/View Controllers/Place.m

```diff
@@ -26,29 +20,57 @@ @implementation Place {
 #pragma mark - Initialization methods
 - (instancetype)initWithDictionary:(NSDictionary *)dictionary {
     self = [super init];
-    dispatch_async(dispatch_get_main_queue(), ^{
-        [self getPlacesClient];
-        if(self) {
-            [self createAllProperties];
-            self.name = dictionary[@"name"];
-            self.address = dictionary[@"formatted_address"];
-            self.coordinates = dictionary[@"geometry"][@"location"];
-            self.iconUrl = dictionary[@"icon"];
-            self.placeId = dictionary[@"place_id"];
-            self.rating = dictionary[@"rating"];
-            self.photos = dictionary[@"photos"];
-            self.types = dictionary[@"types"];
-            [self setPlaceSpecificType];
-            self.unformattedTimes = dictionary[@"opening_hours"];
-            self.locked = NO;
-            self.isHome = NO;
-            self.scheduledTimeBlock = -1;
-            self.timeToSpend = -1;
-            self.hasAlreadyGone = NO;
-            self.isSelected = NO;
-            [self makeScheduleDictionaries];
-        }
-    });
+    //dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
What happened here? hahahah

Reply:
I think this part was yours so I didn't want to change it, but it was causing errors when I tried to run it because of the image part. Since I didn't need images, I just copied all the properties I needed below and kept your part in the comments.